src/udev/udevadm-info.c

```diff
@@ -343,6 +356,8 @@ static int help(void) {
                "       path                     sysfs device path\n"
                "       property                 The device properties\n"
                "       all                      All values\n"
+               "  --property=NAME             Show only properties by this name\n"
+               "  --value                     When showing properties, print only their values\n"
```

Comment:
usually we keep the `--` options aligned vertically, i.e. insert three spaces on the left here please